chain/actors/actor_multisig.go

```diff
@@ -29,11 +29,11 @@ func (msas MultiSigActorState) isSigner(addr address.Address) bool {
 	return false
 }
 
-func (msas MultiSigActorState) getTransaction(txid uint64) *MTransaction {
-	if txid > uint64(len(msas.Transactions)) {
-		return nil
+func (msas MultiSigActorState) getTransaction(txid uint64) (*MTransaction, ActorError) {
+	if 0 == len(msas.Transactions) || txid > uint64(len(msas.Transactions)) {
```

Comment:
That's better, but still not entirely correct, should be:
```suggestion
	if txid >= uint64(len(msas.Transactions)) {
```

Reply:
oh whoops, it was wrong in master D: